packages/visualizations/src/components/Legend/CategoryLegend.svelte

```diff
@@ -0,0 +1,94 @@
+<script lang="ts">
+    import type { CategoryLegend, CategoryItem } from './types';
+
+    export let legendOptions: CategoryLegend;
+    let refinedSeries: string[] = [];
+
+    const isRefined = (item: CategoryItem, series: string[]) =>
+        series.some((color) => item.color === color);
+</script>
+
+{#each legendOptions.items as item, i (item.color)}
```

Comment:
I don't know if this error is really due to this part of code but it seems that we can't have twice the same colour for series... 

![Capture d’écran 2023-06-01 à 09 53 22](https://github.com/opendatasoft/ods-dataviz-sdk/assets/84501475/3280b654-95a2-41df-b5ea-a6be1481c6f3)

It seems also logical to not use colour as the identifier to refine series to avoid refining multiple series in one click... Probably we should give them unique ids